content/api/v18/async_hooks.en.md

```diff
@@ -374,7 +374,7 @@ createHook({
     const eid = executionAsyncId();
     fs.writeSync(
       stdout.fd,
-      `$type($asyncId): trigger: $triggerAsyncId execution: $eid\n`);
+      `${type}(${asyncId}): trigger: ${triggerAsyncId} execution: ${eid}\n`);
```

Comment:
EX: here, is this from the 18.9.0 updates or a bug in the conversion. Linting shouldn't be re-writing docs here, that needs to be addressed upstream

Reply:
Weird, linting should honestly not have applied to the docs for the time being, or at least within code-blocks. Let me check something!

Reply:
I just ran the command to sync the API from upstream (without linting), and the content changes look the same. Maybe this was addressed on upstream.

Reply:
Note.: That our `sync-api` command only fetches and parses the docs into MDX, and doesn't apply any kind of linting, as the docs should be compatible with MDX. The maximum we do is to replace some invalid things, but nothing of the sort as what you have shown in your example, so I bet this was addressed upstream.